lang/cs/Org.Apache.REEF.Client/YARN/YarnREEFParamSerializer.cs

```diff
@@ -123,9 +161,11 @@ internal byte[] SerializeJobArgsToBytes(JobParameters jobParameters, string driv
 
             var avroYarnClusterJobSubmissionParameters = new AvroYarnClusterJobSubmissionParameters
             {
-                yarnJobSubmissionParameters = avroYarnJobSubmissionParameters,
+                //// securityTokenKind and securityTokenService will be deprecated 
```

Comment:
use `TODO` format, reference the JIRA issue, and mention REEF version where it should be removed, e.g.
```C#
// TODO[JIRA REEF-1887] Use SecurityTokenFileName instead of SecurityTokenKindParameter and SecurityTokenServiceParameter. Remove in 0.17
```

Reply:
Done